app/scripts/metamask-controller.js

```diff
@@ -314,7 +315,8 @@ module.exports = class MetamaskController extends EventEmitter {
   }
 
   /**
-   * Returns an api-object which is consumed by the UI
+   * Returns an api-object which is consumed by the UI.
+   * The API object is composed of callback functions.
    *
    * @returns {Object}
```

Comment:
It's usually a good idea to give a description to the `@returns` value just like a `@param`, even if it's a bit redundant. This allows JSDoc parsers to render return value descriptions properly. Maybe something like this:

```js
/**
 * Composes an object containing background API methods. The
 * object is mostly callback functions and is consumed by the UI.
 * 
 * @returns {Object} Object containing API functions
 **/
getApi () {
    // ...
}
```